torch_xla/csrc/lowering_context.cpp

```diff
@@ -67,11 +80,24 @@ class HloMetadataSetter {
     metadata.set_op_name(absl::StrCat(op_name_prefix, op_type));
 
     if (!nmeta.frame_info.empty()) {
```

Comment:
Here we are attempting to correct the following 3 fields:
* source_file
* source_line
* stack_frame_id

Instead of having custom computation here, we should have those passed in.

The location of this change should be inside of of the if in line 69.

Reply:
basically the changes in this function should read something along the lines of:
* if there are custom metadata: pass it down to HLO; else: do what it used to do before. No custom computations should happen here.